src/app/page/cabinet/userDecks/user.decks.component.html

```diff
@@ -146,7 +146,10 @@ <h4 class="modal-title">{{dialogName}}</h4>
                 <input type="text" formControlName="price"
                        [class.error]="deckGroup.controls['price'].touched && !deckGroup.controls['price'].valid"/>
                 <div class="is-invalid error-text" *ngIf="deckGroup.controls['price'].hasError('pattern')">
-                  Enter positive value
+                Enter positive value
+              </div>
+                <div class="is-invalid error-text" *ngIf="deckGroup.controls['price'].hasError('max')">
+                  Please specify a positive number from 0 to 999 999
```

Comment:
There is a validation min(1) on the server side. Min is 0 or 1?

Reply:
Thanks, I fixed it to 1, because when the price is 0 I send null to the back-end.